### DISCUSSION TITLE
Centralized helm source

### DISCUSSION THREAD

**Discussion by Mpluya** [fluxcd/flux2#1151] [Q&A] [answered]
We have a centralized gitrepo for all our helm charts. We defined a helm source in flux-system. We also started following the flux2 guide for multi-tenancy. As we onboard tenants, we were hoping we could reference the 1 centralized helm source running in flux-system.

Q: Is the intent for a helm source to just be namespace scoped? Is there any possibility to allow or permit reading helm sources that are stood up by operators in flux-system?

**Comment by stefanprodan** [comment, marked as answer]
Yes you can modify the RBAC of a tenant and give them access to a named resource.

**Reply by Mpluya** [reply]
thank you @stefanprodan . We will give this a shot. I appreciate your quick response!

**Reply by stefanprodan** [reply]
Also in the Kyverno policy you may want to allow the shared namespace https://github.com/fluxcd/flux2-multi-tenancy/blob/main/infrastructure/kyverno-policies/flux-multi-tenancy.yaml#L23
